privateSBOMExchange/src/scripts/Redaction.py

```diff
@@ -241,9 +46,9 @@ def main():
     #SBOM_file_name=str(args.sbom)
  
     #  path of submodules data 
-    in_the_lab_spdx="../sbom_data/bom-shelter/in-the-lab/spdx-popular-containers/data"
-    in_the_wild_cyclonedx="../sbom_data/bom-shelter/in-the-wild/cyclonedx"
-    in_the_wild_spdx = '../sbom_data/bom-shelter/in-the-wild/spdx' 
+    in_the_lab_spdx="../../sbom_data/bom-shelter/in-the-lab/spdx-popular-containers/data"
+    in_the_wild_cyclonedx="../../sbom_data/bom-shelter/in-the-wild/cyclonedx"
+    in_the_wild_spdx = '../../sbom_data/bom-shelter/in-the-wild/spdx' 
 
 
     SBOM_file_name="julia.spdx.json"
```

Comment:
Same with this and any hardcoded filenames, these should all be passed in via a config file or CLI args.